service/src/main/java/com/epam/esm/service/impl/CertificateServiceImpl.java

```diff
@@ -98,16 +98,18 @@ public  CertificateDto update(CertificateDto certificateDto, long id) {
         if(uniqCertificate != null && uniqCertificate.getId() != id){
             throw new IllegalArgumentException("Certificate name is not uniq " + certificateDto.getName());
         }
-        boolean modify = updateData(giftCertificate, certificateDto);
-        if(modify){
+        if(updateData(giftCertificate, certificateDto)){
```

Comment:
public  CertificateDto update(CertificateDto certificateDto, long id) - logic of the method is incorrect

Reply:
Couldn't you explain more?